extension/src/background/helpers/migration.ts

```diff
@@ -105,20 +110,20 @@ export const migrateTrustlines = async ({
     console.error(e);
   }
 
-  await server.submitTransaction(builtChangeTrustTx);
+  await submitTx({ server, tx: builtChangeTrustTx });
```

Comment:
whenever we submit a tx, let's use this helper to make sure to retry it in case of 504